regression/pubsub.js

```diff
@@ -28,8 +29,20 @@ var Subscription = require('../lib/pubsub/subscription.js');
 
 var pubsub = gcloud.pubsub();
 
+function generateTopicName() {
+  return 'test-topic-' + uuid.v4();
+}
+
+function generateSubName() {
+  return 'test-subscription-' + uuid.v4();
+}
```

Comment:
Instead of using names like `topic1` and `topic2`, I made quick helper methods to generate unique names.